applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -73,7 +73,23 @@ def GetDefaultParameters(cls):
             "bfecc_number_substeps" : 10,
             "distance_reinitialization" : "variational",
             "distance_smoothing" : false,
-            "distance_smoothing_coefficient" : 1.0
+            "distance_smoothing_coefficient" : 1.0,
+            "mass_conservation_correction_settings": {
+                "mass_conservation_check"                : false,
+                "perform_corrections"                    : true,
+                "correction_frequency_in_time_steps"     : 20,
+                "write_to_log_file"                      : true,
+                "log_file_name"                          : "mass_conservation.log"
+            },
+            "distance_modification_settings": {
+                "apply_distance_modification"            : true
+                "distance_threshold"                     : 0.01,
+                "continuous_distance"                    : true,
+                "check_at_each_time_step"                : false,
+                "avoid_almost_empty_elements"            : true,
+                "deactivate_full_negative_elements"      : true,
+                "recover_original_distance_at_each_step" : false
```

Comment:
```suggestion
                "apply_distance_modification"            : true
                "distance_threshold"                     : 1.0e-4,
                "continuous_distance"                    : true,
                "check_at_each_time_step"                : true,
                "avoid_almost_empty_elements"            : false,
                "deactivate_full_negative_elements"      : false
```
These are the suitable defaults for the two-fluids. About the default tolerance, I'd put it smaller.

Reply:
Removed for this PR.